### DISCUSSION TITLE
Can dosxyznrc use a created pegs4 file?

### DISCUSSION THREAD

**Discussion by phantasuson** [nrc-cnrc/EGSnrc#801] [Questions] [answered]
I created a new pegs4 data file and tried to load on dosxyznrc. What dose the pop mean?
![image](https://user-images.githubusercontent.com/80371548/143855924-b8225b08-fad0-408e-b5ed-ef4d69029386.png)

**Comment by ftessier** [comment, marked as answer]
This means that the filename of the pegs file you created (located under `$EGS_HOME/pegs4/data`) is the same as a pegs file distributed with EGSnrc (located under `$HEN_HOUSE/pegs/data`). It warns you that _your_ file will be selected. This is probably what you want, but to avoid confusion I suggest you select a different filename for your file, for example you could add a `my_` prefix. You don't need to recreate the pegs file, just rename it.
